tests/atlas_search_/test_search.py

```diff
@@ -0,0 +1,818 @@
+import unittest
+from collections.abc import Callable
+from functools import wraps
+from time import monotonic, sleep
+
+from django.db import connection
+from django.db.models import Q
+from django.db.models.query import QuerySet
+from django.db.utils import DatabaseError
+from django.test import TransactionTestCase, skipUnlessDBFeature
+from pymongo.operations import SearchIndexModel
+
+from django_mongodb_backend.expressions import (
+    CompoundExpression,
+    SearchAutocomplete,
+    SearchEquals,
+    SearchExists,
+    SearchGeoShape,
+    SearchGeoWithin,
+    SearchIn,
+    SearchMoreLikeThis,
+    SearchPhrase,
+    SearchRange,
+    SearchRegex,
+    SearchScoreOption,
+    SearchText,
+    SearchVector,
+    SearchWildcard,
+)
+
+from .models import Article, Location, Writer
+
+
+def wait_until_index_ready(collection, index_name, timeout: float = 30, interval: float = 0.5):
```

Comment:
30 seconds is a long time. It would be nice to explain if it really could take that long.